lib/src/mount.rs

```diff
@@ -57,3 +57,24 @@ pub(crate) fn mount(dev: &str, target: &Utf8Path) -> Result<()> {
         [dev, target.as_str()],
     )
 }
+
+/// If the fsid of the passed path matches the fsid of the same path rooted
+/// at /proc/1/root, it is assumed that these are indeed the same mounted
+/// filesystem between container and host.
+/// Path *should* not include a leading '/'. It will be stripped.
```

Comment:
I'm fine with this as is but it actually seems simpler to me to just let it be an absolute path (because in fact we are calling it that way just above :smile: ) and rely on the `strip_prefix` to make it relative for the hostpath.